spec/unit/ClientSpec.php

```diff
@@ -626,4 +626,184 @@ function it_receives_an_exception_when_the_api_returns_500(){
 
     }
 
+    //----------------------------------------------------------------------------------------------------------
+    // Template lookups (GETs) with expected success
+
+    function it_generates_the_expected_request_when_looking_up_a_template(){
+
+        //---------------------------------
+        // Test Setup
+
+        $id = '35836a9e-5a97-4d99-8309-0c5a2c3dbc72';
```

Comment:
this is just a random ID i copied from further up the page, and [hopefully] holds no relevance in the real world - just used it for this unit test to prove it gets passed through into the URL